site/docs/v1/tech/lambdas/nintendo-reconcile.adoc

```diff
@@ -0,0 +1,68 @@
+---
+layout: doc
+title: Nintendo Reconcile
+description: An overview of the Nintendo Reconcile lambda
+---
+
+:sectnumlevels: 0
+
+== Nintendo Reconcile lambda
+[NOTE]
+====
+This IdP is still under active development, but is still not code complete.
+====
+
+When an Nintendo identity provider is used to complete a federated login request FusionAuth will use the email address to reconcile user. You may optionally utilize a lambda to customize the user and user registration during this authentication event.
+
+When you create a new lambda using the FusionAuth UI we will provide you an empty function for you to implement.
+
+If you are using the API to create the lambda you will need to ensure your function has the following signature:
+
+[source,javascript]
+----
+function reconcile(user, registration, idToken) {
+  // Lambda code goes here
+}
+----
+
+This lambda must contain a function named `reconcile` that takes three parameters. The parameters that the lambda is passed are:
+
+* `user` - the FusionAuth User object
+* `registration` - the FusionAuth UserRegistration object
+* `idToken` - the JSON payload returned by the Nintendo Token Info API
+// TODO 👆 check above for accuracy
+
+The two FusionAuth objects are well documented here in the link:/docs/v1/tech/apis/users/[User API] and link:/docs/v1/tech/apis/registrations/[Registration API] documentation. The `idToken` may contain various user claims depending upon the user's Nintendo configuration.
+
+== Assigning the lambda
+
+Once a lambda is created, you may assign it to the Nintendo identity provider in the IdP configuration.
+
+Navigate to [breadcrumb]#Settings -> Identity Providers# and select your existing Nintendo configuration or click [breadcrumb]#Add provider# and select Nintendo if it has not yet been configured.
+
+== Default lambda
+
+A default Nintendo reconcile lambda is available in FusionAuth that may be used or modified. The default Nintendo lambda function is documented below.
```

Comment:
We don't ship a default, but when you click 'add' in the UI - you will see the function body below as you have documented. Only certain Lambda types ship with a default.